### ISSUE TITLE
JSON loader configured via meta not functioning

### ISSUE THREAD

**Issue by ghempton** [systemjs/systemjs#1559]
See an example here: https://github.com/ghempton/systemjs-builder-test/tree/json-loader (after checking out run `testem` to test in a browser).

With a configuration such as:

```javascript
SystemJS.config({
  paths: {
    "npm:": "jspm_packages/npm/",
    "app/": "src/"
  },
  browserConfig: {
    "baseURL": "/"
  },
  packages: {
    "app": {
      "format": "register",
      "meta": {
        "*.json": {
          "loader": "json"
        }
      }
    }
  }
});
```

Loading a file with a `.json` extension results in a syntax error.

**Comment by guybedford** [comment]
Thanks, fixed in https://github.com/systemjs/systemjs/commit/2105705a7af99d67541d8a07ab9e4c3e4a8ee015.

**Comment by guybedford** [comment]
Released in 0.20.5.
